package-lock.json

```diff
@@ -5,14 +5,14 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "typescript-runtime-type-benchmarks",
```

Comment:
Odd that this went away?

Reply:
@moltar I may have installed against an old version of NPM (had been running node 12 for the tests), will revert and update NPM.